src/libcommon/utility/utility.cpp

```diff
@@ -619,13 +622,16 @@ QString CommonUtility::getRelativePathFromHome(const QString &dirPath) {
 
 size_t CommonUtility::maxPathLength() {
 #if defined(_WIN32)
-    Poco::Util::WinRegistryKey key(R"(HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\FileSystem)", true);
-    bool exist = key.exists("LongPathsEnabled");
-    if (exist && key.getInt("LongPathsEnabled")) {
-        return MAX_PATH_LENGTH_WIN_LONG;
-    } else {
-        return MAX_PATH_LENGTH_WIN_SHORT;
+    if (_maxPathWin == 0) {
```

Comment:
```suggestion
    static size_t maxPathWin = 0;
    if (maxPathWin == 0) {
```
The initialization will be done on the first call to the function only.